sunpy/io/special/asdf/tests/helpers.py

```diff
@@ -0,0 +1,30 @@
+from io import BytesIO
+
+import asdf
+
+
+def roundtrip_object(obj, version=None):
```

Comment:
This can be removed when we depend on asdf 2.9+

```suggestion
# TODO: Remove this file when depending on asdf 2.9+

def roundtrip_object(obj, version=None):
```